src/components/FormElements/MultiPicker.tsx

```diff
@@ -57,6 +62,19 @@ export const MultiPicker = ({
             })
           : null}
       </>
+      <div>
+        <Button
+          disabled={value.length === limit}
+          size="md"
+          variant="tertiary"
```

Comment:
Should we make this button `secondary` to match other multi-pickers?

![image](https://github.com/stellar/laboratory/assets/7346473/d21edbde-20d9-463f-bc04-d0a98f1b0640)